packages/frontend-app-api/src/wiring/createApp.tsx

```diff
@@ -278,6 +280,7 @@ export function createApp(options?: {
             >
               {/* TODO: set base path using the logic from AppRouter */}
               <BrowserRouter>
+                <RouteTracker routeObjects={routeInfo.routeObjects} />
```

Comment:
This is inside the AppRouter in the legacy application. I believe once SignInPage is migrated we will be able to pull it into a new AppRouter provider.

Reply:
Yep I've also got a feeling this will be restructured at bit at that point 👍

Reply:
Moved to the CoreRouter extension.